src/Mgate/SuiviBundle/Resources/views/Etude/Buttons/Prospect.html.twig

```diff
@@ -2,11 +2,11 @@
  # Input : prospect
  # use : {% include "MgateSuiviBundle:Etude:Buttons/Prospect.html.twig" with  {'prospect':  etude.prospect} %}
  #}
-<div class="btn-group" style="width: 220px;">
-    <a title="{{ prospect.nom }}" class="btn btn-warning" href="{{ path('MgatePersonne_prospect_voir', {'id': prospect.id}) }}" style="width : 60%; text-align: left; overflow: hidden;"><i class="icon-user icon-white"></i> {{ prospect.nom }}</a>
+<div class="btn-group" style="width: 100%; min-width: 220px; max-width:500px;">
```

Comment:
In some of our projects, the name of the client is too long too be displayed in 220px.
Put 100% to have more place to display it when it's possible (to fit the table). Put a limit to 500px, because on "Vue CA" the buttons became really too long.